### PR TITLE
Running

### PR DESCRIPTION
**Pull request by MikeLordx**

### CODE REVIEW

File: ProyectoSemestral3/Assets/Scripts/InventoryScripts/InventoryManager.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+public class InventoryManager : MonoBehaviour
+{
+    public static InventoryManager Instance;
+    public List<Item> items = new List<Item>();
```
Comment (Horothenic) [inline ProyectoSemestral3/Assets/Scripts/InventoryScripts/InventoryManager.cs:7]: Is missing the `_`.

File: ProyectoSemestral3/Assets/Scripts/InventoryScripts/InventoryManager.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    public Transform _itemContent;
+    public InventoryView _inventoryViewerItemPrefab;
```
Comment (Horothenic) [inline ProyectoSemestral3/Assets/Scripts/InventoryScripts/InventoryManager.cs:9]: Why public?

File: ProyectoSemestral3/Assets/Scripts/InventoryScripts/InventoryManager.cs [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+            Destroy(item.gameObject);
+        }
+
+        foreach (var item in items)
+        {
+            InventoryView itemView = Instantiate(_inventoryViewerItemPrefab, _itemContent);
+            itemView.Initialize(item);
+        }
+    }
+
```
Comment (Horothenic) [inline ProyectoSemestral3/Assets/Scripts/InventoryScripts/InventoryManager.cs:39]: The extra space is after the last `}`

File: ProyectoSemestral3/Assets/Scripts/InventoryScripts/InventoryView.cs [hunk 4]
```diff
@@ -0,0 +1,14 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+public class InventoryView : MonoBehaviour
+{
+    public Image _itemIcon;
+    public Text _itemName;
```
Comment (Horothenic) [inline ProyectoSemestral3/Assets/Scripts/InventoryScripts/InventoryView.cs:7]: Should be `[SerializeField] private` and have a default value.

File: ProyectoSemestral3/Assets/Scripts/InventoryScripts/Item.cs [hunk 5]
```diff
@@ -0,0 +1,10 @@
+using UnityEngine;
+
+[CreateAssetMenu(fileName = "New Item", menuName = "Item/Create New Item")]
+public class Item : ScriptableObject
+{
+    public int id;
+    public string itemName;
+    public string value;
+    public Sprite icon;
```
Comment (Horothenic) [inline ProyectoSemestral3/Assets/Scripts/InventoryScripts/Item.cs:9]: Add default value.

Comment (Horothenic) [reply ProyectoSemestral3/Assets/Scripts/InventoryScripts/Item.cs:9]: And `_`

File: ProyectoSemestral3/Assets/Scripts/InventoryScripts/ItemController.cs [hunk 6]
```diff
@@ -0,0 +1,9 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ItemController : MonoBehaviour
+{
+    public Item item;
+
```
Comment (Horothenic) [inline ProyectoSemestral3/Assets/Scripts/InventoryScripts/ItemController.cs:8]: Remove extra space and why public? also `_`, and initialize it.

File: ProyectoSemestral3/Assets/Scripts/InventoryScripts/ItemController.cs [hunk 7]
```diff
@@ -0,0 +1,9 @@
+using System.Collections;
+using System.Collections.Generic;
```
Comment (Horothenic) [inline ProyectoSemestral3/Assets/Scripts/InventoryScripts/ItemController.cs:2]: Remove unused libraries.

File: ProyectoSemestral3/Assets/Scripts/PlayerScripts/PlayerMovement.cs [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+public class PlayerMovement : MonoBehaviour
+{
+    [SerializeField] private CharacterController _controller;
+    [SerializeField] private Transform cam;
+    [SerializeField] private float _speed = 6f;
+    private float _runningSpeed = 12f;
+    private float _speedDecrement = 6f;
+    [SerializeField] private float _turnTime = 0.1f;
+    [SerializeField] float _directionDeadZone = 0.1f;
+    float _turnVelocity;
```
Comment (Horothenic) [inline ProyectoSemestral3/Assets/Scripts/PlayerScripts/PlayerMovement.cs:14]: Have all `[SerializeField] private` together, all private together, and check that all variables have default value, `_` and access modifier.

File: ProyectoSemestral3/Assets/Scripts/PlayerScripts/PlayerMovement.cs [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+        if (Input.GetKeyUp(KeyCode.LeftShift))
+        {
+            _speed = _speedDecrement;
+        }
+        
+    }
+
+
+
+}
```
Comment (Horothenic) [inline ProyectoSemestral3/Assets/Scripts/PlayerScripts/PlayerMovement.cs:42]: Add spaces between code blocks (the `if`s) and remove extra spaces below.

**Review by Horothenic — CHANGES_REQUESTED** [review]

File: ProyectoSemestral3/Assets/Scripts/InventoryScripts/InventoryView.cs (hunk 4 above)
Comment (MikeLordx) [reply ProyectoSemestral3/Assets/Scripts/InventoryScripts/InventoryView.cs:7]: Done

File: ProyectoSemestral3/Assets/Scripts/InventoryScripts/Item.cs (hunk 5 above)
Comment (MikeLordx) [reply ProyectoSemestral3/Assets/Scripts/InventoryScripts/Item.cs:9]: Done

File: ProyectoSemestral3/Assets/Scripts/InventoryScripts/ItemController.cs (hunk 6 above)
Comment (MikeLordx) [reply ProyectoSemestral3/Assets/Scripts/InventoryScripts/ItemController.cs:8]: Done

**Review by Horothenic — APPROVED** [review]

**Pull request merged**